notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb

```diff
@@ -444,7 +368,7 @@
         "id": "0wguBZeKp93_"
       },
       "source": [
-        "If you are using ***Vertex AI Workbench managed notebooks instance***, every cell which starts with \"#@bigquery\" will be a SQL Query. If you are using Vertex AI Workbench user managed notebooks instance or Colab it will be a markdown cell."
+        "If you are using ***Vertex AI Workbench managed notebooks instance***, Identify cells starting with \"#@bigquery\" as SQL queries. If you are using Vertex AI Workbench user managed notebooks instance or Colab it will be a markdown cell."
```

Comment:
RE: "If you are using Vertex AI Workbench user managed notebooks instance or Colab it will be a markdown cell."
- should add link to the migration doc, since "Vertex AI Workbench user-managed notebooks" is being deprecated. 
    - Here's the link: https://cloud.google.com/vertex-ai/docs/workbench/user-managed/migrate-to-instances